DeepLinkKit/Router/DPLDeepLinkRouter.m

```diff
@@ -137,6 +137,19 @@ - (BOOL)handleURL:(NSURL *)url withCompletion:(DPLRouteCompletionBlock)completio
 }
 
 
+- (nullable DPLDeepLink *)deepLinkForURL:(nonnull NSURL *)url {
```

Comment:
Would this instead of lines 120 - 127 would do it?

```
deepLink = [self deepLinkForURL:url];
isHandled = [self handleRoute:route withDeepLink:deepLink error:&error];
```